sys/xtimer/xtimer_core.c

```diff
@@ -198,7 +198,7 @@ int _xtimer_set_absolute(xtimer_t *timer, uint32_t target)
     DEBUG("timer_set_absolute(): now=%" PRIu32 " target=%" PRIu32 " offset=%" PRIu32 "\n",
           now, target, offset);
 
-    if (offset <= XTIMER_BACKOFF) {
+    if (offset <= xtimer_ticks_from_usec(XTIMER_BACKOFF).ticks32) {
```

Comment:
Can you use a define for this? This will always be a constant value right?

Reply:
I think it will be optimized already anyway. Its a static inline function.

Reply:
XTIMER_BACKOFF is independently defined for some boards, as well as the clocks so this shouldn't be constant for every possible instance, am i wrong?

Reply:
> XTIMER_BACKOFF is independently defined for some boards

Yes, but still constant value. Anyway it was more a question about compiler optimization but as @cladmi pointed out it should be optimized out sinze it is inline. So I'll resolve de conversation.